### PR TITLE
Add migration started step

### LINKED ISSUE #91852: Migration Instructions: Migration Started Step
### Details

After the user completes all the steps, redirect them to a "Migration Started" step with a call to action for Automattic for Agencies.

The navigation and file for the component were already created in this commit: https://github.com/Automattic/wp-calypso/pull/92387/commits/992eab5d24eec8bd9eeb2cc5b18224927e2c578d#diff-d9890bf9823cb09810951292977c288efa8b271733992be696ec304677b3f85aR6

See p9Jlb4-c8w-p2#comment-11896.

Screenshot 2024-06-17 at 14 34 46

### Checklist

- [x] Created the new Migration Started step.
- [x] The flow navigates to this step when the user completes the migration steps.
- [ ] Tests created.

### Related

https://github.com/Automattic/dotcom-forge/issues/7703

### PR DESCRIPTION
**Pull request by renatho**
Related to #91852

## Proposed Changes

* Add the content and styles to the Migration Started step.

## Testing Instructions

* Make sure you have the `migration-flow/new-migration-instructions-step` feature flag enabled.
* Create a new site with the Import intent.
* Follow all the steps until the Migration Started step.
* Check that it works well in different screen sizes.

## Screenshots

Screenshot 2024-07-16 at 15 19 30

Screenshot 2024-07-16 at 15 19 22

cc @fditrapani to validate the responsive design for small screens, where I hid the image to have enough space for the button.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-started/index.tsx [hunk 1]
```diff
@@ -1,16 +1,67 @@
...
+					{ translate(
+						'The new Automattic for agencies will help you manage all the sites in one place to streamline your workload and work smarter.'
+					) }
+				</p>
+				<ExternalLink
+					className="migration-started-card__cta"
+					href="https://automattic.com/for-agencies/"
+					target="_blank"
+					onClick={ () =>
+						recordTracksEvent( 'calypso_onboarding_site_migration_started_agencies_cta_click' )
```
Comment (renatho) [inline client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-started/index.tsx:38]: Here we also register an event to be able to track numbers of clicks on the CTA.

**Review by m1r0 — APPROVED** [review]
Looks good! 🙌

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16332656

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @renatho for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16332656).

**Pull request merged**